### PR TITLE
refactor(components): stabilize appearance

### PR DESCRIPTION
**Pull request by yinonov**
fixes many discrepancies accross components and re-unifies deviations where outline-fill-text model is applied

### CODE REVIEW

**Comment by yinonov** [comment]
in relevance to #805

File: libs/components/src/lib/accordion-item/accordion-item.scss [hunk 1]
```diff
@@ -93,6 +93,7 @@
 	}
 
 	.icon {
+		color: var(#{constants.$vvd-color-neutral-600});
```
Comment (olaf-k) [inline libs/components/src/lib/accordion-item/accordion-item.scss:96]: the icon has now a different color to the item's text, is it on purpose?
image

Comment (olaf-k) [reply libs/components/src/lib/accordion-item/accordion-item.scss:96]: same for `nav-disclosure` / `nav-item`

**Review by olaf-k — COMMENTED** [review]
The placeholders in the number|text fields are the same color as the user input, it probably should not.
image

**Review by olaf-k — COMMENTED** [review]
Labels (for checkbox, number/text field, etc.) also don't change color when the widget is disabled. Is that on purpose?

File: libs/components/src/lib/accordion-item/accordion-item.scss (hunk 1 above)
Comment (yinonov) [reply libs/components/src/lib/accordion-item/accordion-item.scss:96]: yes. on purpose. that's the original design and still need to be addressed again as the question was raised. it's by design spec and has been deviated from accidentally

**Comment by yinonov** [comment]
> The placeholders in the number|text fields / text-area are the same color as the user input, it probably should not. image

nice catch! resolved

**Comment by yinonov** [comment]
> 

resolved. readonly should still be interactive

**Comment by yinonov** [comment]
> Labels (for checkbox, number/text field, etc.) also don't change color when the widget is disabled. Is that on purpose?

they shouldn't. the change is deliberate

**Review by olaf-k — APPROVED** [review]
We can discuss the radio/cursors/readonly state outside of this PR. I'll have a look at the latter.

**Comment by rachelbt** [comment]
> Not related to your changes but in scope of appearance consistency:
> 
> * number/text fields & text-area lack the `not-allowed` cursor in disabled state - https://github.com/Vonage/vivid-3/issues/811
> * the readonly state for checkbox/radio/switch should probably not keep the `pointer` cursor - not sure - need to be checked.

**Review by olaf-k — APPROVED** [review]

**Pull request merged**